src/pagination/pagination.ts

```diff
@@ -7,41 +7,46 @@ import {getValueInRange, toInteger, toBoolean} from '../util/util';
 @Component({
   selector: 'ngb-pagination',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [`
+    button.page-link {
+      outline: 0;
```

Comment:
I think that we all know that there are no perfect solutions (that is - or at least we can't think of them atm... ) - longish discussion in #142 is the evidence. So I think that we are at the point where we need to choose _something_ - even if not perfect we need to choose the least bad approach. And we need to do something for alpha.3 since links without `href` are styled black.

So, this is the decision time. I can see the following options:
* this PR
* variant of this where we remove outline only from the focused buttons (to be tested)
* what I've proposed in https://github.com/ng-bootstrap/ng-bootstrap/issues/142#issuecomment-231587330 - empty `href` with prevent default

I can't think of anything else atm. So please, let's have a discussion about what _can_ be done. Either pick one of the above or propose an alternative, otherwise it will be hard progress on this one.